Makefile

```diff
@@ -64,24 +65,15 @@ build-tnf-tool:
 
 # Cleans up auto-generated and report files
 clean:
-	go clean
-	rm -f ./cnf-certification-test/cnf-certification-test.test
-	rm -f ./cnf-certification-test/cnf-certification-tests_junit.xml
-	rm -f ./cnf-certification-test/claim.json
-	rm -f ./cnf-certification-test/claimjson.js
-	rm -f ./cnf-certification-test/results.html
-	rm -f ./cnf-certification-test/cnf-certification-tests_junit.xml
-	rm -f ./tnf
-	rm -f latest-release-tag.txt
-	rm -f release-tag.txt
-	rm -f jsontest-cli
-	rm -f test-out.json
-	rm -f cover.out
-	rm -f claim.json
-	rm -f all-releases.txt
+	go clean && rm -f all-releases.txt cover.out claim.json cnf-certification-test/claim.json \
+		cnf-certification-test/claimjson.js cnf-certification-test/cnf-certification-test.test \
+		cnf-certification-test/cnf-certification-tests_junit.xml \
+		cnf-certification-test/results.html jsontest-cli latest-release-tag.txt \
+		release-tag.txt test-out.json tnf
 
 # Run configured linters
 lint:
+	checkmake Makefile
```

Comment:
@line 111, it would be great to add installation for other linters:
# Install linters
install-lint:
	go install github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_VERSION}

Reply:
@edcdavid The problem is they have different install command for different OS, need to think about, will do another PR.